src/mscorlib/model.xml

```diff
@@ -6325,11 +6406,19 @@
       <Member Name="#ctor(System.Char[])" />
       <Member Name="#ctor(System.Char[],System.Int32,System.Int32)" />
       <Member Name="#ctor(System.SByte*)" />
+      <Member Name="#ctor(System.SByte*,System.Int32,System.Int32)" />
+<!--
+      <Member Name="#ctor(System.SByte*,System.Int32,System.Int32,System.Text.Encoding)" />
```

Comment:
To enable this one, you need to remove ifdef around `COMString::StringInitSBytPtrPartialEx` in `ecalllist.h`